src/UI/DatePicker.elm

```diff
@@ -0,0 +1,414 @@
+                        ((px 44), 20)
+                    else
+                        ((px 32), 16)
+    in
+    Element.el
+        [ Element.width (fillPortion 7)
+        , Element.centerX
+        , if isSelected then
+            Palette.blue700 |> Palette.toBackgroundColor
+
+          else
+            Palette.genericWhite |> Palette.toBackgroundColor
+        , Element.mouseOver
+            [ Palette.gray200 |> Palette.toBackgroundColor
+            ]
+        , Element.Events.onClick <| externalToMsg <| Selected day.date
+        , if day.date == today then
+            Element.inFront (Element.el [ Element.centerX, Element.alignBottom ] (Element.text "."))
+
+          else
+            Element.inFront Element.none
+        , Border.rounded 6
+        ]
+    <|
+        Element.column
+            [ Element.height cellHeight
+            , Element.centerX
+            , if day.isCurrentMonth && not isSelected then
+                Palette.blue800 |> Palette.toFontColor
+
+              else if day.isCurrentMonth && isSelected then
+                Palette.genericWhite |> Palette.toFontColor
+
+              else
+                Palette.gray500 |> Palette.toFontColor
+            , Font.size fontSize
+            ]
+            [ Element.el
+                [ Element.centerY
+                ]
+              <|
+                Element.text <|
+                    String.fromInt (Date.day day.date)
+            ]
+
+
+drawCol : String -> Element msg
+drawCol day =
+    Element.el [ Element.width (fillPortion 7), Element.centerX, Element.centerY ] <|
+        Element.el [ Element.centerX, Font.size 12, Palette.gray700 |> Palette.toFontColor ]
+            (Element.text <| day)
+
+
+header : RenderConfig -> (Msg -> msg) -> Property msg -> Element msg
+header renderConfig externalMsg model =
+    let
+        label =
+            Date.format "MMMM" model.current ++ " " ++ (String.fromInt <| Date.year model.current)
+
+        iconSize = if RenderConfig.isMobile renderConfig then
+                                Size.medium
+                    else
+                        Size.small
+    in
+    Element.row [ Element.width fill ]
+        [ Button.fromIcon (Icon.chevronLeft "previous")
+            |> Button.cmd (externalMsg PreviousMonth) Button.clear
+            |> Button.withSize iconSize
+            |> Button.renderElement renderConfig
+            |> Element.el [ alignLeft ]
+        , label
+            |> Text.subtitle1
+            |> Text.withColor Palette.gray800
+            |> Text.renderElement renderConfig
+            |> Element.el [ Element.centerX ]
+        , Button.fromIcon (Icon.chevronRight "previous")
+            |> Button.cmd (externalMsg NextMonth) Button.clear
+            |> Button.withSize iconSize
+            |> Button.renderElement renderConfig
+            |> Element.el [ alignRight ]
+        ]
+
+
+weekHeader : List String
+weekHeader =
+    [ "Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun" ]
```

Comment:
We'll have to make this depend on `RenderConfig` so we can use the translatable terms...